src/frontend/qt_sdl/Platform.cpp

```diff
@@ -271,6 +271,23 @@ FileHandle* OpenFile(const std::string& path, FileMode mode)
     }
 }
 
+void MakeLocalDirectory(const std::string& dir)
+{
```

Comment:
To match the behavior of `OpenLocalFile`, this should use absolute paths as-is.

Reply:
good catch